### ISSUE TITLE
FW: Past due invoices - Checking Account number length

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#1201]
In GitLab by @kdjstudios on Oct 19, 2018, 12:10

**Submitted by:** "Kimberly Gagner" <kim.mckellar@answernet.com>

**Helpdesk:** http://www.servicedesk.answernet.com/profiles/ticket/2018-10-19-32597/conversation

**Server:** All

**Client/Site:** All

**Account:** All

**Issue:** 

Please read the clients issue below when attempting to make a payment on our portal.

> I have been trying to make a payment the past two months and it will not accept my checking account numbers for either account.  It says it must only be 9 digits.  Well, both accounts more than 9 digits in their account numbers.  Last time I tried I figured it was a glitch so I just figured I would try again later.  I completely forgot.  I am trying again now and the same thing keeps happening - both offices - both checking accounts.
> 
> What would you like me to do?  We have never had a problem paying by check before.

Full Email: [original_message__17_.html](/uploads/c0aa24e28f217db4b86d60119486396c/original_message__17_.html)

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Oct 19, 2018, 12:12

@smasih @pchaudhary @tim.traylor @amishra 

Please look into this, as this is preventing cleints from being able to pay their bills. 

NOTE: I am marking this as a bug, because if I recall correctly @tim.traylor had previously mentioned this before and it should have been resolved. I can not find the ticket but if I do I will link it.

EDIT: Found @tim.traylor comment: http://gitlab.aavaz.biz/AnswerNet/SABilling/issues/639#note_26980 Also found might be related to #653 as these appear to be the same accounts.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Oct 22, 2018, 01:13

@kdjstudios 

> NOTE: I am marking this as a bug, because if I recall correctly @tim.traylor had previously mentioned this before and it should have been resolved. I can not find the ticket but if I do I will link it.


Yes, you are right and we did fix it but only on sab/ops portal and forgot to do it on the client's portal.     
Currently, this validation issue is fixed on both portals.  

Routing Number Limit: [+minimum 9+]..[-maximum 9-]

Account Number Limit: [+minimum 1+]..[-maximum 17-]

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Oct 22, 2018, 08:41

@smasih 

May you please clarify, why this was not fully completed in the original mention?

I wrote:

> The support team has updated the ability to allow for larger account numbers. May you please confirm it is working for both you and your client.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Oct 22, 2018, 22:32

@kdjstudios 

When this came up we did a hot fix for this change, and we're trying to get off the code asap because users were already using it. So, we forgot to make the change on the client portal. 

I understand what your concern and, there is no right clarification for this. So, form this we have learned that we need to be an extra careful while doing the hot-fixes and need to revisit the fix next morning just to see for possible areas where it could go wrong.
